wp-content/themes/workingnyc/views/home.twig

```diff
@@ -45,14 +45,14 @@
     <div class="bg-image-grid hidden tablet:grid grid-cols-2 animated fadeIn">
```

Comment:
Can you delete the div starting at line 45? That's only used for the background image, but the background image should instead be added inside the div that starts at line 9.

Reply:
If I delete the div then the round image is not displaying.

Reply:
You should move the image to a different div - the one that starts at line 9